src/Components/TableComponents/TextInputBasic.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+
+import PropTypes from 'prop-types';
+
+import {
+  Flex,
+  HelperText,
+  HelperTextItem,
+  Icon,
+  TextInput,
+  TextInputGroup,
+  TextInputGroupMain,
+} from '@patternfly/react-core';
+
+import { CheckCircleIcon, InProgressIcon } from '@patternfly/react-icons';
+
+export const TextInputGroupBasic = ({
+  textValue,
+  setTextValue,
+  verifiedStatus,
+  isLoading,
+}) => {
+  return (
+    <Flex direction={{ default: 'column' }}>
+      <TextInputGroup>
+        <TextInputGroupMain
+          value={textValue}
+          onChange={(_event, value) => setTextValue(value)}
+        />
+        <TextInput
+          value={textValue}
+          type="text"
+          validated={!isLoading && verifiedStatus === 'error' && verifiedStatus}
+          customIcon={
+            (isLoading && <InProgressIcon />) ||
+            (!isLoading && verifiedStatus === 'success' && (
+              <Icon status="success">
+                <CheckCircleIcon />
+              </Icon>
+            ))
+          }
```

Comment:
Adding the Fragment at the end here will keep the text box from resizing each time you type in a text box for the first time.

```suggestion
          customIcon={
            (isLoading && <InProgressIcon />) ||
            (!isLoading && verifiedStatus === 'success' && (
              <Icon status="success">
                <CheckCircleIcon />
              </Icon>
            )) || <></>
          }
```